guides/rif-relay/deployment.md

```diff
@@ -0,0 +1,314 @@
+      | SmartWalletRelayVerifier          | 0x5159345aaB821172e795d56274D0f5FDFdC6aBD9 |
+      | Custom Smart Wallet Contracts =================================================|
+      | CustomSmartWallet                 | 0x7557fcE0BbFAe81a9508FF469D481f2c72a8B5f3 |
+      | CustomSmartWalletFactory          | 0x0e19674ebc2c2B6Df3e7a1417c49b50235c61924 |
+      | CustomSmartWalletDeployVerifier   | 0x8901a2Bbf639bFD21A97004BA4D7aE2BD00B8DA8 |
+      | CustomSmartWalletRelayVerifier    | 0x1Af2844A588759D0DE58abD568ADD96BB8B3B6D8 |
+      | Testing Contracts =============================================================|
+      | SampleRecipient                   | 0xCd5805d60Bbf9Afe69a394c2BDa10F6Dae2c39AF |
+      | TestToken                         | 0x726ECC75d5D51356AA4d0a5B648790cC345985ED |
+      |===================================|============================================|
+      ```
+
+      As we can see we have two sets of Smart Wallets with their own verifiers. This is because each verifier use the factory to do deploy and relay validations. For testing purposes we will be using the Smart Wallet Contracts. 
+
+  - 2) Collector Contract
+    * In principle, any number of Collector contracts can be deployed and used. This does not automatically occur alongside the deployment of the rest of the Relay contracts during the execution of migrations since usage of the Revenue Sharing feature is optional. For an overview of the Collector contract please see [the corresponding architecture document section](/rif/relay/architecture/#collector).
+
+      Before deploying a Collector contract please make sure that:
+      1. The token chosen for this contract matches the one used for relayed transaction fees. **Any tokens other than the one set at the time of the Collector deployment which are sent to the contract will be impossible to retrieve, including the native token**. 
+      2. The owner of the Collector contract has been chosen (be it a multisig contract address, a regular contract address or an EOA). This does not need to match the address deploying the contract, but please note that if there are no means of signing Collector contract function calls with the owner address **all revenue sharing funds will be permanently locked in the contract**.
+      3. Partners and their shares have been defined correctly. This means that beneficiary shares must collectively add up to 100. As with the previous point, any tokens sent to an address without its private key held by its beneficiary will be lost. For an example of a structurally valid revenue shares definition see [this deploy collector task input sample](https://github.com/rsksmart/rif-relay-contracts/blob/PP-94/revenue-sharing-model/deploy-collector.input.sample.json).
+
+    * To deploy the Collector contract we will be using the [RIF Relay Contract](https://github.com/rsksmart/rif-relay-contracts) and [Safe contracts v1.2.0](https://github.com/gnosis/safe-contracts/tree/v1.2.0).
```

Comment:
updated

Reply:
Not sure about the changes, but I meant to specify that although we'll be mentioning the safe contracts, the important thing is that the owner of the collector should be able to call the withdraw method, so it can be either an EOA or any other account (i.e.: multisig).

Reply:
updated